src/main/java/com/spotify/ffwd/http/model/v2/Batch.java

```diff
@@ -0,0 +1,82 @@
+/*-
+ * -\-\-
+ * FastForward API
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.ffwd.http.model.v2;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+/**
+ * This class represents a batch of points.
+ * This implementation of the batch uses {@link Value} as the point value.
+ */
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@EqualsAndHashCode(of = {"commonTags", "commonResource"})
+public class Batch {
+    private final Map<String, String> commonTags;
+    private final Map<String, String> commonResource;
+    private final List<Point> points;
+
+    /**
+     * JSON creator.
+     */
+    @JsonCreator
+    public static Batch create(
+        @JsonProperty("commonTags")
```

Comment:
just a nitpick thing, would be easier to read if they're all one liners

Reply:
Agreed, but too many params and annotations, I cannot fit everything on a single line.